thriftbp/server_middlewares.go

```diff
@@ -380,3 +383,64 @@ func RecoverPanic(name string, next thrift.TProcessorFunction) thrift.TProcessor
 		},
 	}
 }
+
+// PrometheusMetricMiddleware returns middleware to track Prometheus metrics specific to the Thrift service.
+// It emits the following prometheus metrics:
```

Comment:
nit: add a blank line (`//`) between these 2 lines, and do some line-wrapping on the long lines in the comment.